lib/game.rb

```diff
@@ -0,0 +1,36 @@
+class Game
+  attr_reader :player1, :player2
+  def initialize
+    @player1 = player1
+    @player2 = player2
+  end
```

Comment:
Like I mentioned in your `war_or_peace_runner.rb` file, I would refactor this `initialize` method like this:

```ruby
def initialize(player1, player2)
  @player1 = player1
  @player2 = player2
end
```